### PR TITLE
Store remote votes URI

### PR DESCRIPTION
**Pull request by ClearlyClaire**
This is useful if we ever plan to support remote users deleting their votes.

### CODE REVIEW

File: app/lib/activitypub/tag_manager.rb [hunk 1]
```diff
@@ -35,6 +35,8 @@ def uri_for(target)
       account_status_url(target.account, target)
     when :emoji
       emoji_url(target)
+    when :vote
+      [uri_for(target.account), '#votes/', target.id].join
```
Comment (Gargron) [inline app/lib/activitypub/tag_manager.rb]: Well, if they are going to be stored, might generate a real path for them?

Comment (ThibG) [reply app/lib/activitypub/tag_manager.rb]: we could, but I'm not sure if that is useful, much like follows

Comment (Gargron) [reply app/lib/activitypub/tag_manager.rb]: I don't disagree, but the follows &co are not in this method :confused: Perhaps you could move them here then?

Comment (ThibG) [reply app/lib/activitypub/tag_manager.rb]: Changed it to use the same logic as `Follow` and cie

**Review by Gargron — APPROVED** [review]

**Pull request merged**